### ISSUE TITLE
"core_1.Version is not a constructor" on ng serve

### ISSUE THREAD

**Issue by pierznj** [degt/angular2-blog-example#1]
I clone, `npm install` and then `ng serve` and get `core_1.Version is not a constructor`. Some versioning issue, but I'm using the provided package.json... Any help?

**Comment by degt** [comment]
Hello!. Which angular-cli version are you running'?

**Comment by degt** [comment]
angular-cli 1.0.0-rc.4 will fail.
angular-cli 1.0.0-beta.15 works fine.

**Comment by pierznj** [comment]
Thanks for the reply. I tried `npm install --save-dev angular-cli@1.0.0-beta.15` to ensure I was running the correct version, but `ng` still seems to run the globally installed version of angular-cli, which is the latest. So I uninstalled angular-cli globally, deleted node_modules and ran `npm install` again. No changes made to the repo at all. I could then `ng serve`, but eventually get a bunch of errors similar to: `ERROR in [default] <path>/angular2-blog-example/node_modules/@types/jasmine/index.d.ts:40:37
A parameter initializer is only allowed in a function or constructor implementation.`
